src/components/Culr/culr.component.js

```diff
@@ -61,35 +61,37 @@ export async function getUserAttributesFromCulr(user = {}, createCulrAccountAgen
 
   if (responseCode === 'OK200') {
     attributes.accounts = response.result.Account;
-    const {
-      blocked,
-      userPrivilege,
-      municipalityNumber,
-      municipalityAgencyId
-    } = await getUserInfoFromBorchk(response.result, user);
-    attributes.blocked = blocked || false;
-    attributes.userPrivilege = userPrivilege || [];
-    attributes.municipalityNumber = municipalityNumber;
-    attributes.municipalityAgencyId = municipalityAgencyId;
     attributes.culrId = response.result.Guid || null;
-    return attributes;
-  }
-
+    const userInfo = await getUserInfoFromBorchk(response.result, user);
+    addUserInfoToAttributes(attributes, userInfo);
```

Comment:
Jeg tror du skal have en if her som du har i line 70, for at være sikker på at der er data i userInfo

Reply:
Har flyttet if'en ind i addUserInfoToAttributes funktionen